docs/docs/providers/42.md

```diff
@@ -13,7 +13,9 @@ https://api.intra.42.fr/apidoc/guides/web_application_flow
 
 ## Configuration
 
-https://profile.intra.42.fr/oauth/applications/new
+https://profile.intra.42.fr/oauth/applications/new  
+The callback uri has to be  
+DOMAIN.TLD/api/auth/callback/42-school
```

Comment:
```suggestion
```
We need to remove this as it's already going to be explained here: https://github.com/nextauthjs/next-auth/pull/4065

Reply:
Thanks for the hard work!